mtags/src/main/scala-2/scala/meta/internal/pc/SemanticTokenProvider.scala

```diff
@@ -307,8 +305,10 @@ final class SemanticTokenProvider(
         case df: cp.MemberDef if df.pos.isRange =>
           (annotationChildren(df) ++ df.children)
             .foldLeft(
-              nodes :+ NodeInfo(df, df.namePos)
+              if (nodes(NodeInfo(df, df.namePos))) nodes
+              else nodes + NodeInfo(df, df.namePos)
```

Comment:
In scala 2.12.14 `SortedSet` updates the element, in 2.13.10 it doesn’t, so this is needed in 
```scala
class Foo(<<bar>>: Int)
```
because there are two symbols on `bar`: `val` in class and parameter in `init` method. We want to choose the first one